packages/utilities/psammead-styles/src/fonts.test.js

```diff
@@ -0,0 +1,18 @@
+// Test to verify functionality of overriding base font URL
+
+import * as fonts from './fonts';
+
+Object.values(fonts).forEach(font => {
```

Comment:
Can you wrap this all (lines 5 - 18) in a `describe` block please with a description of `Psammead Styles - Fonts`